tests/bootstrap_tests/test_configuration.py

```diff
@@ -251,6 +251,7 @@ def setUp(self):
         # thread startup and shutdown, we only want to ensure they are invoked.
         for mocked_method in [
             "tank.bootstrap.import_handler.CoreImportHandler.swap_core",
+            "tank.bootstrap.cached_configuration.CachedConfiguration._ensure_core_local",
```

Comment:
From here on out I will be providing links to jump to relevant changes to the unit tests because about 40+ files were deleted.

Reply:
Click [here] to jump.